### PR TITLE
wip: Try to get qtwebkit building on 23.08

### PR DESCRIPTION
**Pull request by jbruechert**

### CODE REVIEW

**Comment by bbhtt** [comment]
Is this still wip or do you plan to do something?

bot, build

File: io.qt.qtwebkit.BaseApp.json [hunk 1]
```diff
@@ -39,12 +52,8 @@
         },
         {
           "type": "archive",
-          "url": "https://github.com/qtwebkit/qtwebkit/releases/download/qtwebkit-5.212.0-alpha4/qtwebkit-5.212.0-alpha4.tar.xz",
-          "sha256": "9ca126da9273664dd23a3ccd0c9bebceb7bb534bddd743db31caf6a5a6d4a9e6"
-        },
-        {
-          "type": "patch",
-          "path": "qtwebkit-build.patch"
+          "url": "https://github.com/movableink/webkit/archive/3cfcdcf38a520a22a9e86bd4b8ffa17f2ee2e1aa.tar.gz",
```
Comment (bbhtt) [inline io.qt.qtwebkit.BaseApp.json:55]: Why was the source switched?

Comment (jbruechert) [reply io.qt.qtwebkit.BaseApp.json:55]: This is the fork that the old repository now points to: https://github.com/qtwebkit/qtwebkit

**Comment by jbruechert** [comment]
> Is this still wip or do you plan to do something?

I have pretty much abandoned this, as the project I needed this for has ported away from QtWebKit.

File: io.qt.qtwebkit.BaseApp.json (hunk 1 above)
Comment (bbhtt) [reply io.qt.qtwebkit.BaseApp.json:55]: thanks

**Comment by bbhtt** [comment]
> > Is this still wip or do you plan to do something?
> 
> I have pretty much abandoned this, as the project I needed this for has ported away from QtWebKit.

Seems like a few apps are still using Qtwebkit, I need it for notepaddqq, but trojita is also using it. It's on life support, but let's merge this, so that they can at least update to 23.08.

**Comment by bbhtt** [comment]
Branch created https://github.com/flathub/io.qt.qtwebkit.BaseApp/tree/branch/5.15-23.08

Build started https://buildbot.flathub.org/#/builders/6/builds/87612

**Pull request closed without merging**